pkg/services/hcloud/loadbalancer/loadbalancer.go

```diff
@@ -352,6 +357,8 @@ func statusFromHCloudLB(lb *hcloud.LoadBalancer, hasNetwork bool, log logr.Logge
 	for _, target := range lb.Targets {
 		switch target.Type {
 		case hcloud.LoadBalancerTargetTypeServer:
+			p := &target.Type
+			fmt.Println("p", p)
```

Comment:
@janiskemper there is a debug statement: fmt.Println()

Reply:
merci